app/views/layouts/shared/header/_header_landing.html.slim

```diff
@@ -0,0 +1,49 @@
+.header-navbar.navbar-expand-sm.navbar.navbar-horizontal.navbar-light.navbar-shadow.border-grey.border-lighten-2.navbar-brand-center.space-between.land-font.fixed-top.pl-5 data-nav="brand-center"
+  /! Horizontal menu content
+  .horizontal-menu.navbar-container.ml-5
+    = link_to root_path do
+      = image_tag("logo1.png", alt: 'Sales Assistant', id: 'logo')
+  .horizontal-menu.navbar-container.ml-0
+    ul.nav.navbar-nav data-menu="menu-navigation"
+      li.nav-item
+        = link_to "HOME", home_path, class:"nav-link font-medium-2"
+      li.nav-item
+        = link_to "ABOUT US", about_us_path, class:"nav-link font-medium-2"
+      li.nav-item
+        = link_to "PRICING", pricing_path, class:"nav-link font-medium-2"
+      li.nav-item
+        = link_to "CONTACTS", contacts_path, class:"nav-link font-medium-2"
+  - if user_signed_in?
+    .horizontal-menu.navbar-container

```

Comment:
don't we have such code already in other layout?
extract to partial.
this dropdown when user is logged in.
Also "arrow" next to username in this layout looks not aligned well.

Reply:
edited the arrow and made a partial for that bit of a header
![screenshot from 2018-01-08 22-21-30](https://user-images.githubusercontent.com/23340152/34692262-3d33b16e-f4c8-11e7-9e27-c3b43b6dcfdc.png)